File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprstrn.java
// EXPECTED: 20
// Reference emit for CKPRSTRN (TAKE → phase 20).
public final class CkprstrnRef {
  public static void main(String[] args) {
    char entry = 'T';
    int phase = switch (entry) {
      case 'I' -> 0;
      case 'T' -> 20;
      case 'C' -> 30;
      case 'R' -> 40;
      default -> 99;
    };
    System.out.printf("%02d%n", phase);
  }
}

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_ckprstrn.cs
// EXPECTED: 20
// Reference emit for CKPRSTRN (TAKE → phase 20).
using System;
public static class CkprstrnRef {
  public static void Main() {
    char entry = 'T';
    int phase = entry switch {
      'I' => 0,
      'T' => 20,
      'C' => 30,
      'R' => 40,
      _ => 99,
    };
    Console.WriteLine(phase.ToString("00"));
  }
}
